### ISSUE TITLE
undefined method `call' for Array

### ISSUE THREAD

**Issue by stereodenis** [nevir/rubocop-rspec#41]
```
bundle exec rubocop spec/models -a
Inspecting 47 files
....

4 files inspected, no offenses detected
undefined method `call' for #<Array:0x007f9603fcc740>
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/cop/corrector.rb:46:in `block in rewrite'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/cop/corrector.rb:45:in `each'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/cop/corrector.rb:45:in `rewrite'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/cop/team.rb:90:in `autocorrect_one_cop'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/cop/team.rb:65:in `autocorrect'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/cop/team.rb:35:in `inspect_file'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/runner.rb:135:in `inspect_file'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/runner.rb:106:in `block in do_inspection_loop'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/runner.rb:100:in `loop'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/runner.rb:100:in `do_inspection_loop'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/runner.rb:78:in `process_file'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/runner.rb:53:in `block in inspect_files'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/runner.rb:51:in `each'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/runner.rb:51:in `inspect_files'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/runner.rb:30:in `run'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/lib/rubocop/cli.rb:26:in `run'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/bin/rubocop:13:in `block in <top (required)>'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/2.2.0/benchmark.rb:303:in `realtime'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.32.1/bin/rubocop:12:in `<top (required)>'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/bin/rubocop:23:in `load'
/Users/stereodenis/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/bin/rubocop:23:in `<main>'
```

**Comment by geniou** [comment]
It looks like the error is caused in `rubocop` not `rubocop-rspec`.

**Comment by backus** [comment]
Not sure if this helps, but it seems like this is due to the autocorrect api changing in bbatsov/rubocop/pull/1833

**Comment by backus** [comment]
Changing

```ruby
def autocorrect(node)
  @corrections << lambda do |corrector|
    corrector.replace(node.loc.expression, 'described_class')
  end
end
```

to

```ruby
def autocorrect(node)
  lambda do |corrector|
    corrector.replace(node.loc.expression, 'described_class')
  end
end
```

fixes it for me

**Comment by bquorning** [comment]
Fixed in #39.

**Comment by geniou** [comment]
@bquorning is right.. so I'm closing thus and we hope to have #39 merged soon. Sorry for the delay.
